client/rest-high-level/src/main/java/org/elasticsearch/client/MachineLearningClient.java

```diff
@@ -426,6 +428,44 @@ public void getBucketsAsync(GetBucketsRequest request, RequestOptions options, A
                 Collections.emptySet());
      }
 
+    /**
+     * Gets the categories for a Machine Learning Job.
+     * <p>
+     * For additional info
+     * see <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-get-category.html">
+     *     ML GET categories documentation</a>
+     *
+     * @param request  The request
+     * @param options  Additional request options (e.g. headers), use {@link RequestOptions#DEFAULT} if nothing needs to be customized
```

Comment:
I know some methods add the `@throws` and others don't. I think we should still add the `@throws` entry even if it is a checked exception.

I have not been consistent about this myself either :(.

Reply:
@dimitris-athanasiou may have a different opinion around this ^, So I will defer to him.

Reply:
You're right @benwtrent, we've been dropping the `@throws` clause in some of the methods in the client. We'll need to revisit and add them. I'll make a note to do that.